CHANGES.rst

```diff
@@ -4,7 +4,10 @@ Changelog
 9.2 (unreleased)
 ----------------
 
-- Nothing changed yet.
+Other changes
++++++++++++++
+
+- Deprecated `--result-log` was removed in pytest 6.1.0.
```

Comment:
Nit: can we change the wording a bit to something like 'Deprecate `--result-log`, as it was removed in  pytest 6.1.0 )'